tests/generation/test_configuration_utils.py

```diff
@@ -224,6 +256,411 @@ def test_generation_mode(self):
         config = GenerationConfig()
         self.assertEqual(config.get_generation_mode(assistant_model="foo"), GenerationMode.ASSISTED_GENERATION)
 
+    def test_serialize_generation_sequence_bias(self):
```

Comment:
```suggestion
class GenerationConfigSerializationTest(unittest.TestCase):
    def test_serialize_generation_sequence_bias(self):
```
These tests deserve their own category 💛

Reply:
agree, done